site/en/blog/fedcm-origin-trial/index.md

```diff
@@ -710,6 +634,13 @@ The `login()` call takes the following properties:
   </tr>
 </table>
 
+The browser handles sign-up and sign-in use cases differently depending on the
+existence of `approved_clients` in the response from [the accounts list
+endpoint](#accounts-list-endpoint). The browser will only display [the RP's
+privacy policy and terms of service](#client-metadata-endpoint) in the dialog if
+`approved_clients` isn't provided and the user hasn't previously signed up for
```

Comment:
"isn't provided" -> "isn't provided OR does not include the RP's client_id AND the user hasn't..."